src/app/information/information.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+
+
+@Component({
+  selector: 'app-information',
+  templateUrl: './information.component.html',
+  styleUrls: ['./information.component.sass']
+})
+export class InformationComponent implements OnInit {
+
+  name: FormGroup;
+  email: FormGroup;
+  phone: FormGroup;
+  inputValue = '';
+  showImage = false;
+  toggleName = true;
+  toggleEmail = false;
+  togglePhone = false;
+  toggleImage = false;
+  toggleSubmit = false;
+  buttonStatus = true;
+  person = {
+    name: '',
+    email: '',
+    phone: '',
+    image: {},
+  };
+
+  constructor(
+    private router: Router
+  ) { }
+
+  goToResultsPage() {
+    const user = JSON.stringify(this.person);
+    localStorage.setItem('user', user);
+    this.router.navigate(['/results']);
+  }
+
+  inputName() {
```

Comment:
don't need handle input values on typing
you can take that from `form.values`
describe above how create a form